code/modules/reagents/reagent_containers/food/drinks/drinkingglass.dm

```diff
@@ -557,6 +555,57 @@
 					reagents.trans_to(P, reagents.total_volume)
 					spawn(1)
 						qdel(src)
+				if(SCIENTISTS_SERENDIPITY)
+					if(reagents.get_reagent_amount(SCIENTISTS_SERENDIPITY)<10) //You need at least 10u to get the tech bonus
+						icon_state = "scientists_surprise"
+						name = "\improper Scientist's Surprise"
+						desc = "There is as yet insufficient data for a meaningful answer."
+					else
+						icon_state = SCIENTISTS_SERENDIPITY
+						name = "\improper Scientist's Serendipity"
+						desc = "Knock back a cold glass of R&D."
+						origin_tech = "materials=7;engineering=3;plasmatech=2;powerstorage=4;bluespace=6;combat=3;magnets=6;programming=3"
```

Comment:
Isnt this the recommended research amount for an average round?

Reply:
It is, but it's really hard to find a specific fossiled plant and make a bar drink out of it before R&D is done anyway. If it does happen, though, there's your serendipity.